portfolio/src/main/webapp/profiles.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Profile Links </title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h1>Ethan's Profiles</h1>
+    <p>Here is a list of links to my online profiles. </p>
+    <a href="https://www.linkedin.com/in/ethanrigby/">LinkedIn</a> </p>
```

Comment:
Here and on the next few lines you have some dangling `</p>` tags. Not sure if you meant to wrap the <a> tags in a paragraph or not.

Reply:
They were all on the same line. I just added the <p> before because I didn't see I forgot them

Reply:
Alright sounds good. There is just one `</p>` missing after instagram now.